tests/requirements/base.txt

```diff
@@ -3,14 +3,15 @@ django-filer>=1.4.0
 djangocms-picture>=2.1.0
 djangocms-link>=2.2.1
 django-polymorphic>=2.0.3
-django-treebeard<4.5
 Pillow
 html5lib>=0.999999999
 
 # other requirements
+coverage
```

Comment:
are we sure to run a unpinned version of packages>

Reply:
For some things, absolutely.

Coverage is the thing running the test suite, so if there's an issue with it, we'd know at the point code is pushed.

It's also only a package that's involved with the dev side of things. 

There's enough to keep track of with versions of packages that manage functionality that certainly need some degree of pinning. Things like coverage are safe imo